src/StreamJsonRpc.Tests/PerfTests.cs

```diff
@@ -151,7 +151,9 @@ public async Task BurstInvokeMessages(Stream serverStream, Stream clientStream)
         }
     }
 
+#pragma warning disable SA1414 // Tuple types in signatures should have element names
```

Comment:
💭 This one seems like an easy fix to just add the names

Reply:
Easy fix, but IMO inappropriate since there are literally no semantic names we could add over the ones that ValueTuple provides. We just have two streams.